requirements.txt

```diff
@@ -70,14 +69,15 @@ requests==2.13.0          # via requests-aws4auth
 six==1.10.0               # via bcrypt, bleach, cryptography, html5lib, packaging, python-dateutil
 sqlalchemy==1.1.4
 statsd==3.2.1
-transaction==2.1.2        # via pyramid-mailer, pyramid-tm, repoze.sendmail, zope.sqlalchemy
+transaction==2.1.2
 translationstring==1.3    # via colander, deform, pyramid
 unidecode==0.4.19         # via python-slugify
 urllib3==1.16             # via elasticsearch
-venusian==1.0             # via pyramid
+venusian==1.0
+webencodings==0.5.1       # via html5lib
```

Comment:
This one line appears to be a new addition; not quite sure what’s going on there.